src/fio/fioEngine.ts

```diff
@@ -107,13 +114,15 @@ export class FioEngine extends CurrencyEngine<FioTools> {
     opts: EdgeCurrencyEngineOptions,
     tpid: string
   ) {
-    super(env, tools, walletInfo, opts)
+    const safeWalletInfo = asSafeFioWalletInfo(walletInfo)
+    const fioPrivateKeys = asFioPrivateKeys(walletInfo.keys)
+    super(env, tools, safeWalletInfo, opts)
```

Comment:
The constructor won't get the private keys anymore, so this is no good. Are we still waiting to merge Matthew's FIO refactor before we can fix this?

Reply:
Yes, this is still blocked by Matt's work.